src/igvfd/tests/test_upgrade_model_set.py

```diff
@@ -5,3 +5,9 @@ def test_model_set_upgrade_1_2(upgrader, model_set_v1):
     value = upgrader.upgrade('model_set', model_set_v1, current_version='1', target_version='2')
     assert value['schema_version'] == '2'
     assert 'description' not in value
+
+
+def test_model_set_upgrade_1_2(upgrader, model_set_v3):
```

Comment:
Test name is duplicated (probably should be 3_4)

Reply:
fixed